### PR TITLE
Add link to TwigQI in docs "You might also be interested in" section

### PR DESCRIPTION
**Pull request by drjayvee**
As proposed by Fabien last Thursday at Evolve. Merci beaucoup!

### CODE REVIEW

**Review by ruudk — APPROVED** [review]
Good idea, thanks for adding!

File: doc/templates.rst [hunk 1]
```diff
@@ -69,6 +69,10 @@ You might also be interested in:
 * `Twig Language Server`_: provides some language features like syntax
   highlighting, diagnostics, auto complete, ...
 
+* `TwigQI`_: an extension which analyzes your templates for common bugs during compilation.
```
Comment (fabpot) [inline doc/templates.rst]:
```suggestion
* `TwigQI`_: an extension which analyzes your templates for common bugs during compilation
```

**Comment by drjayvee** [comment]
Thanks for the review, Fabien.

**Comment by fabpot** [comment]
Thank you @drjayvee.

**Pull request closed without merging**